packages/web-shared/components/FeatureFeed/Features/EventBlockFeature.styles.js

```diff
@@ -0,0 +1,83 @@
+import styled from 'styled-components';
+import { withTheme } from 'styled-components';
+import { themeGet } from '@styled-system/theme-get';
+import { rgba } from 'polished';
+
+import { system } from '../../../ui-kit/_lib/system';
+
+export const Container = withTheme(styled.ul`
+  display: flex;
+  flex-direction: column;
+  margin-bottom: ${themeGet('space.base')};
+  color: ${themeGet('colors.text.primary')};
+  border-radius: ${themeGet('radii.xl')};
+  background-color: ${themeGet('colors.fill.paper')};
+  box-shadow: ${themeGet('shadows.medium')};
```

Comment:
This shadow seems excessive to me, but I don't like the `low` option either
![Screenshot 2024-03-01 at 1 55 07 PM](https://github.com/ApollosProject/apollos-embeds/assets/72768221/5e4e91cc-e07b-45b7-90b3-60fe21feb4d1)

Admin's shadow is much more subtle
![Screenshot 2024-03-01 at 1 56 35 PM](https://github.com/ApollosProject/apollos-embeds/assets/72768221/ce1c0804-2582-415c-a28c-5d44e62382cd)